### ISSUE TITLE
add Facbeook Infer to CI?

### ISSUE THREAD

**Issue by rgerhards** [rsyslog/rsyslog#2540]
see http://fbinfer.com/docs/getting-started.html

This might be especially useful as Coverity Scan for open source projects seems to be dead (at least is so for quite a while now) --> https://twitter.com/tatsuhiro_t/status/973559750938959872

**Comment by kortemik** [comment]
coverity has become available again

**Comment by rgerhards** [comment]
@kortemik indeed, we are also using it for quite a while again (rsyslog QA automatically uses it when available). But infer is still interesting as an addition, albeit at a lower priority.
